web/angular-wallet/src/app/store/store.service.ts

```diff
@@ -27,10 +27,6 @@ export class StoreService {
     * Called on db start
     */
```

Comment:
Removed storing blocks... it was pointless and error-prone (testnet and mainnet blocks co-mingling).